### DISCUSSION TITLE
how to get current screen size via theme ?

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#5447] [Q&A]
it is necessary to obtain information about which device is currently selected and, for example, for a modal window, specify default props depending on the device
![image](https://github.com/mantinedev/mantine/assets/83704092/57e92adf-1193-44c3-91cb-1071aa894eb5)

**Comment by rtivital** [comment]
Do not use defaultProps for that, use Styles API to apply styles instead.
